README.md

```diff
@@ -122,6 +122,9 @@ These are not implementations of lit-html itself but rather community extensions
 - [`<lottie-player>`](https://github.com/LottieFiles/lottie-player) - Web Component for easily embedding and playing Lottie animations.
 - [`<model-viewer>`](https://github.com/GoogleWebComponents/model-viewer) - A web component for rendering interactive 3D models.
 - [`<rapi-doc>`](https://github.com/mrin9/RapiDoc) - Web Component to view OpenAPI 3.0 & Swagger 2.0 Spec.
+- [`<code-block>`](https://github.com/justinribeiro/code-block) - Web component that displays colorfully formatted code with Prism.js and LitElement.
+- [`<stl-part-viewer>`](https://github.com/justinribeiro/stl-part-viewer) - LitElement web component that utilizes Three.js to display an STL model file.
+- [`<barcode-reader>`](https://github.com/justinribeiro/barcode-reader) - Web component that reads barcodes via the Shape Detection API via a Web Worker, powered by LitElement.
```

Comment:
The experimental status of this component (as it is described in [README](https://github.com/justinribeiro/barcode-reader#experimental)) might mean it is not the best candidate to be mentioned in the awesome list. So I would consider not adding it here for now.

Reply:
Sorry, that's my bad. BarcodeDetector got bumped back to M83 (was supposed to go in M82), so once that lands I'll remove that section and re-PR it in.